detox/src/devices/android/Emulator.js

```diff
@@ -24,8 +24,15 @@ class Emulator {
   }
 
   async boot(emulatorName) {
-    const headless = argparse.getArgValue('headless') ? '-no-window' : '';
-    const cmd = `-verbose -gpu host -no-audio ${headless} @${emulatorName}`;
+    const cmd = _.compact([
+        '-verbose',
+        '-gpu',
+        'host',
```

Comment:
Change to `gpu auto` for better support. Current command breaks on some machines